Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -97,6 +101,14 @@ public bool MakeTraitor(EntityUid traitor, TraitorRuleComponent component, bool
 
         _antag.SendBriefing(traitor, GenerateBriefing(component.Codewords, code, issuer), null, component.GreetSoundNotification);
 
+        // Send codewords to only the traitor client
+        var color = TraitorFallbackColor; // Fall back to a dark red Syndicate color if a prototype is not found
+        if (_prototypeManager.TryIndex(TraitorRadioChannel, out RadioChannelPrototype? syndieChannel))
+        {
+            color = syndieChannel.Color;
+        }
+        RaiseNetworkEvent(new RoleCodewordEvent(GetNetEntity(mindId), color, "traitor", component.Codewords.ToList()), traitor);
```

Comment:
this should be a method in a server/shared side RoleCodewordSystem rather than raising the network event directly.